tests/test_codebuild_cli.py

```diff
@@ -0,0 +1,204 @@
+import json
+import os
+from pathlib import Path
+from unittest.mock import patch
+
+import boto3
+import pytest
+from click.testing import CliRunner
+from moto import mock_codebuild
+from moto import mock_iam
+from moto import mock_ssm
+from moto import mock_sts
+
+from commands.codebuild_cli import AWS_REGION, codedeploy, create_codedeploy_role, slackcreds
+from commands.codebuild_cli import check_git_url, link_github
+from commands.codebuild_cli import check_service_role
+from commands.codebuild_cli import update_parameter
+from utils.aws import check_aws_conn
+
+@pytest.fixture
+def aws_credentials():
+    """Mocked AWS Credentials for moto."""
+    moto_credentials_file_path = Path(__file__).parent.absolute() / 'dummy_aws_credentials'
+    os.environ['AWS_SHARED_CREDENTIALS_FILE'] = str(moto_credentials_file_path)
+
+
+@pytest.fixture
+def alias_session(aws_credentials):
+    with mock_iam():
+        session = boto3.session.Session()
+        session.client("iam").create_account_alias(AccountAlias="foo")
+        
+        yield session
+
+
+# Not much value in testing these while moto doesn't support `import_source_credentials`` or `list_source_credentials`
+# def test_import_pat():
+#     ...
+# def test_check_github_conn():
+#     ...
+
+
+@mock_iam
+def test_check_service_role_does_not_exist(capfd, aws_credentials):
```

Comment:
In line with the conversation @lgarvey and I are having over here: https://github.com/uktrade/copilot-tools/pull/44#discussion_r1221373977

I wonder if these test methods could be named so as to better express what's actually being tested?

E.g.
```suggestion
def test_check_service_role_throws_error_when_codebuild_role_does_not_exist(capfd, aws_credentials):
```

Reply:
I've tried to use docstrings for this purpose, while keeping the test names a bit terser because people will be typing or copypasting these and it's useful to keep them shortish in that context

Reply:
I prefer more explicit method names for tests, then the output from the test run, especially if there are failures, is more helpful.